### ISSUE TITLE
Setenv does not work properly with non-nebra fleets

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/hm-pktfwd#123]
Originally reported by @hellrc5 here https://github.com/NebraLtd/helium-rak/issues/19#issuecomment-1418269534

```
packet-forwarder Error: Cannot set FREQ variable.
packet-forwarder VARIANT variable is already set as COMP-RAKHM.
packet-forwarder Error: Not all required parameters have been set. Container will be reset after 5 sec.

Interestingly the diagnostics container detects the correct FREQ plan:

diagnostics INFO:hw_diag.utilities.hardware:Found the following LTE Devices: []
diagnostics INFO:root:Frequency: EU868
```

**Comment by shawaj** [comment]
Relates-to: https://github.com/NebraLtd/hm-diag/issues/444
Relates-to: https://github.com/NebraLtd/hm-diag/issues/468

**Comment by MuratUrsavas** [comment]
The best way to fix this, is to make FREQ parameter optional, not mandatory.

**Comment by shawaj** [comment]
Closed by #126
